bluebrain/repo-bluebrain/packages/py-bbp-workflow/package.py

```diff
@@ -27,7 +27,8 @@ class PyBbpWorkflow(PythonPackage):
     depends_on('py-bluepy-configfile',         type='run')
     depends_on('py-voxcell',                   type='run')
     depends_on('py-entity-management',         type='run')
-    depends_on('py-xarray',                    type='run')
+    depends_on('py-xarray@0.18.2',             type='run')
```

Comment:
I see that in `setup.py` there is `xarray~=0.18.0`. To be compatible with any 0.18.x that will be added to spack, maybe this can be changed to the following?
```suggestion
    depends_on('py-xarray@0.18.0:0.18.999',             type='run')
```
or
```suggestion
    depends_on('py-xarray@0.18.0:0.18',             type='run')
```

Reply:
NVM, I see that after xarray 0.18.2 there is already xarray 0.19, so it's unlikely that other 0.18.x will be released.

Reply:
Yeah, and they just switched to CalVer but I'm waiting for some major PRs to be merged to update versions...